test/bad_dtls_test.c

```diff
@@ -548,8 +548,9 @@ static int test_bad_dtls(void)
 
         if (!TEST_true(send_record(rbio, SSL3_RT_APPLICATION_DATA, tests[i].seq,
                                    &tests[i].seq, sizeof(uint64_t)))) {
-            TEST_error("Failed to send data seq #0x%lx (%d)\n",
-                       tests[i].seq, i);
+            TEST_error("Failed to send data seq #0x%08x%08x (%d)\n",
+                       (uint32_t)(tests[i].seq >> 32),
+                       (uint32_t)(tests[i].seq & 0xFFFFFFFF), i);
```

Comment:
> We have %j in BIO_printf...

But testutil.c doesn't use BIO_print, i.e. it doesn't matter what BIO_printf understands...